<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>member_add</class>
 <widget class="QDialog" name="member_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>회원등록</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>101</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>아이디</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>비밀번호</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>닉네임</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>포인트</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>쿠폰</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="vip_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>60</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>vip</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="common_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>일반</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>211</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="id_txt">
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pw_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="name_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="point_txt"/>
    </item>
    <item>
     <widget class="QLineEdit" name="coupon_txt"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="add_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>99</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>등급</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="premium_btn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>premium</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px</string>
   </property>
   <property name="text">
    <string>중복확인</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
